ui/apps/platform/src/Containers/NetworkGraph/NetworkGraphPage.css

```diff
@@ -4,3 +4,8 @@
     --pf-c-page__main-section--PaddingLeft: 0;
     --pf-c-page__main-section--PaddingRight: 0;
 }
+
+.pf-topology-resizable-side-bar {
+    /* 155.2px is the height of the page header + nav header */ 
+    height: calc(100vh - 155.2px);
+}
```

Comment:
You may want to turn on **Insert Final Newline** in your editor.

<img width="807" alt="Screen Shot 2022-11-04 at 1 29 24 PM" src="https://user-images.githubusercontent.com/715729/200038937-82e6576d-4ebf-463e-9fbf-acbed80b5314.png">